src/v2/Apps/Purchase/Components/OrderRow.tsx

```diff
@@ -38,15 +37,21 @@ const StyledImage = styled(Image)`
   width: 50px;
 `
 
+const UnderlineOnHoverLink = styled(RouterLink)`
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+  }
+`
+
 const OrderRow: React.FC<OrderRowProps> = props => {
   const { order } = props
+  const displayState = order.displayState
```

Comment:
This is the main refactor here. We've introduced `displayState` to order in Exchange that makes all those views much more streightforward.